src/components/editor-page/use-notification-test.tsx

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
Should this be part of this PR?

Reply:
Yes. It will be replaced as soon as we have real notification integrations.

Reply:
What he says